articles/stream-analytics/stream-analytics-compatibility-level.md

```diff
@@ -138,11 +138,11 @@ The following major changes are introduced in compatibility level 1.1:
 
 **1.1 level:** CREATE TABLE allows you to specify a strong schema. The Stream Analytics engine validates that the data conforms to this schema. With this model, the command can filter events with NaN values.
 
-### Disable automatic upcast for datetime strings in JSON
+### Disable automatic conversion of datetime strings to DateTime type at ingress for JSON
 
-**1.0 level:** The JSON parser would automatically upcast string values with date/time/zone information to DateTime type and then convert it to UTC. This behavior resulted in losing the timezone information.
+**1.0 level:** The JSON parser would automatically convert string values with date/time/zone information to DATETIME type at ingress so the value will immediately lose original formatting and timezone information. Since this was done at ingress, even if that field was not used in the query it will now be converted into UTC DateTime.
```

Comment:
```suggestion
**1.0 level:** The JSON parser would automatically convert string values with date/time/zone information to DATETIME type at ingress so the value immediately loses its original formatting and timezone information. Since this is done at ingress, even if that field was not used in the query, it is converted into UTC DateTime.
```